experimental/auth-logic/wrappers/rekor_wrapper_test.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package wrappers
+
+import (
+	"testing"
+)
+
+const testRekorLogPath = "experimental/auth-logic/test_data/rekor_entry.json"
+
+func TestRekoLogWrapper(t *testing.T) {
+	logEntryAnon, err := getLogEntryAnonFromFile(testRekorLogPath)
+	if err != nil {
+		t.Fatalf("couldn't parse rekor log entry from path: %s. %v", testRekorLogPath, err)
+	}
+
+	rekordEntry, err := getRekordEntryFromAnon(*logEntryAnon)
+	if err != nil {
+		t.Fatalf("couldn't get rekordEntry from body of logEntryAnon. rekordEntry: %v, logEntryAnon: %v, rekordLogFilePath: %s. err: %v", *rekordEntry, *logEntryAnon, testRekorLogPath, err)
+	}
+
+	_, err = verifyRekordLogSignature(rekordEntry)
```

Comment:
Could you also validate the public key, since that's the main output?

Reply:
It's not the main output. This is just code meant to write the auth logic wrapper. The auth logic wrapper does not need this public key for anything.

Reply:
wait, how is it possible that the public key is not needed for anything? am I missing something?

Reply:
The auth logic wrapper just needs to say that it validated the signature. So it does need the public key just internally to the function `verifyRekordLogSignature`, which only needs to return a bool saying whether the (key, data, signature) triple was valid (basically the result of `ecdsa.VerifyASN1`). Since we are not using the `principals=keys` feature of auth logic, there is no use for this key other than to check that signature.

Reply:
Unless the idea is that we also apply the product team key as an input (to this whole `auth-logic-transparent-release` program) and check that input against the one fetched here?

Reply:
Yeah sorry my wording was poor, I did mean validate the signature, and return the public key (you are right that if the key was invalid in the first place, the signature would not check out -- one hopes)